src/styles/default.ts

```diff
@@ -61,8 +61,8 @@ const defaultStyle = css`
   p,
   li {
     margin: 0.8rem 0;
-
     font-size: 1.6rem;
+    line-height: calc(16px * 1.5);
```

Comment:
Why is this not just 18px?

Reply:
I had an ide of making it more dynamic, having the font-size as a variable and multiplying it by 1.5.
But there was a bit too much work for that in this time frame that we have.
I will remove the calculation.